scraper/README.md

```diff
@@ -0,0 +1,52 @@
+# PermaplanT Scraper
+
+## Requirements
+
+-   nodejs 19.4.0
+-   npm
+
+## Installation and Usage
+
+1. Install dependencies
+
+```shell
+npm install && mkdir data
+```
+
+2. Start the scraper
+
+In this step data is scraped from the source webpage and stored in the `data` folder in a csv format. There are two types of data that can be scraped: `list` with all available plants and `details` for each specific plant. The scraper can be started with the following commands.
+
+Fetch plants list:
+
+```shell
+npm run fetch:list
+```
+
+Fetch details for each plant in the list:
+
+```shell
+npm run fetch:details
+```
+
+Combine both methods with a single command:
+
+```shell
+npm run fetch
+```
+
+3. Insert data into database
```

Comment:
Also needs fixes here. Write that `diesel migration run` must be executed first.

Reply:
I added a note to let the reader know that database should be up-to-date with the latest migrations